ingest/user_de.py

```diff
@@ -170,15 +177,31 @@ def generate_individual_files(col, genes, rest, groups, clean_val, qual):
 
 # final result: individual files for each comparison
 
+def generate_manifest(clean_val, qual):
+    """
+    create manifest file of each comparison in the initial data
+    if the comparison is with rest, rest is omitted and just the type is written
+    """
+    file_names = []
+    for i in clean_val:
+        cleaned_list = [ x for x in i if x != "rest" and x not in qual]
+        file_name = '_'.join(cleaned_list)
```

Comment:
Blocker: please ensure each pairwise comparison line has both groups in their natural sort order.  

For example, if the groups in the comparison are "type_2" and "type_10", the manifest line should list them as 

* type_2     type_10

and not

* type_10     type_2

we'll want to ensure that's so even if the user lists "type_10" before "type_2" in their DE file.  Having the test file contain "type_10" before "type_2" would be a good way to ensure we manually test that.  This would also be a great motivating example to start on automated tests for user DE in Ingest Pipeline!